backend/docker-compose.yml

```diff
@@ -36,7 +36,7 @@ services:
     ports:
       - "3890:389"
   postgresql:
-    image: postgres:9.6
+    image: postgres:13.3
```

Comment:
I think i should remove this change, but there is maybe something to think about if we do use postgresql 13 but have strict support on postgresql > 9.6.